airgun/entities/job_template.py

```diff
@@ -14,6 +15,11 @@ class JobTemplateEntity(BaseEntity):
     def create(self, values):
         """Create new job template"""
         view = self.navigate_to(self, 'New')
+        wait_for(
+            lambda: JobTemplateCreateView(self.browser).is_displayed is True,
```

Comment:
Is this `wait_for` necessary?
Our implementation of `am_i_here` calls view `is_displayed` method, so calling it explicitly shouldn't be required in common cases.

Reply:
This solved the problem in automation, where the test tried to fill in the form before it was displayed, not sure why it's happening in this case.